aptos-move/framework/PROVER-GUIDE.md

```diff
@@ -0,0 +1,38 @@
+This guide collects links to the prover documents and gives some hints to troubleshoot issues when using the prover. 
+
+## Installation
+
+Please refer to the [doc](https://aptos.dev/cli-tools/install-move-prover/).
+
+## Specification 
+
+The specification for Move contracts are written in the Move Specification Language. 
+Please refer to this [doc](https://github.com/aptos-labs/aptos-core/blob/main/third_party/move/move-prover/doc/user/spec-lang.md) 
```

Comment:
Remember to repoint this to the docs @junkil-park is currently porting into the regular docsets

Reply:
This is the doc for introducing the Move Specification Language. I will check with @junkil-park on the location and update the link.